modules/pxp-module-puppet

```diff
@@ -282,6 +282,10 @@ def action_run(input)
         1
       end
 
+      # flush the stdout/stderr before writing the exitcode
+      # file to avoid pxp-agent reading incomplete output
+      $stdout.fsync
```

Comment:
I think it's just clutter then, adding the ```sync = true``` doesn't really help anything - worse it may trick someone to believe that there is something like a working autoflush in ruby.